chainerx_cc/chainerx/backward.cc

```diff
@@ -208,13 +208,15 @@ class BackwardImpl {
             const BackpropId& backprop_id,
             DoubleBackpropOption double_backprop,
             std::unordered_map<ArrayNode*, internal::GradRef> array_node_grad_map,
-            bool retain_grad)
+            bool retain_grad,
+            const absl::optional<float>& loss_scale)
```

Comment:
How about using `float loss_scale` defaults to `1.0`?

Reply:
I thought of it, but I would like to disable some actions such as array tracking if the loss scaling is not set, I didn't want to do the checks as `loss_scale != 1.0` as it doesn't seem safe enough.